### PR TITLE
Update logistics page

### PR DESCRIPTION
**Pull request by llevitis**
I've finished adding instructions for pretty much all the software that we'll be asking people to install for the TrainTrack. Let me know if I've left anything out :) 

The instructions are mostly reused from Ross Markello's setup instructions for Montreal BHS 2020. https://school.brainhackmtl.org/setup/

### CODE REVIEW

**Comment by Remi-Gau** [comment]
Do we want to point to this google spreasheet?

[**Project checklist**](https://docs.google.com/spreadsheets/d/1SY-Hnhc-bPX0RA47icVukUVVJ-BrX9ykPFQUBY-U1go/edit?usp=sharing):
this is a spreadsheet to help the project helpers making sure that each project is on track and submitted everything.

**Comment by Remi-Gau** [comment]
I will update the mattermost section with the relevant help desk channels but let's not let that stop us from merging

File: logistics.md [hunk 1]
```diff
@@ -203,14 +261,349 @@ these steps:
...
+
+###### **Git**
+
+You most likely already have Git installed.
+To check if you do:
+- On Debian -  type `sudo apt-get install git` inside the terminal
+- On Fedora - type `sudo yum install git` inside the terminal.
+
+###### **VSCode**
+Go to https://code.visualstudio.com/ and click the download button, then run the `.exe` file.
```
Comment (Remi-Gau) [inline logistics.md]: I think this is the windows section and  not the linux section, no?

Comment (llevitis) [reply logistics.md]: woops will fix!

**Comment by Remi-Gau** [comment]
I think we should merge this soon and then update as needed, no?

One thing we might need to add are the instructions for installing datalad though I am tempted to simply point to the datalad ~~bible~~ handbook. What do you think?

**Comment by llevitis** [comment]
> I think we should merge this soon and then update as needed, no?
> 
> One thing we might need to add are the instructions for installing datalad though I am tempted to simply point to the datalad ~bible~ handbook. What do you think?

i added links to the handbook :)

File: logistics.md (hunk 1 above)
Comment (llevitis) [reply logistics.md]: those are the instructions i found here: https://school.brainhackmtl.org/setup/

File: logistics.md [hunk 2]
```diff
@@ -203,14 +270,371 @@ these steps:
...
+
+###### **Git**
+
+You most likely already have Git installed.
+To check if you do:
+- On Debian -  type `sudo apt-get install git` inside the terminal
+- On Fedora - type `sudo yum install git` inside the terminal.
+
+###### **VSCode**
+Go to [visual studio code website](https://code.visualstudio.com/) and click the download button, then run the `.exe` file.
```
Comment (Remi-Gau) [inline logistics.md:282]: I found this on there https://school.brainhackmtl.org/setup/

I think that unless you use Wine you can't run a `.exe` on thelinux, right?

```suggestion
    Go to https://code.visualstudio.com/ and click the download button for either the .deb (Ubuntu, Debian) or the .rpm (Fedora, CentOS) file.
    Double-click the downloaded file to install VSCode. (You may be prompted to type your administrator password during the install).
```

File: logistics.md [hunk 3]
```diff
@@ -203,14 +270,371 @@ these steps:
...
+Leave all the defaults during the installation with the following exception:
+Please make sure the box labelled “Register Code as an editor for supported file types” is selected.
```
Comment (Remi-Gau) [inline logistics.md:284]: I found this on there https://school.brainhackmtl.org/setup/

I think that unless you use Wine you can't run a `.exe` on thelinux, right?

```suggestion
    Go to the [visual studio code website](https://code.visualstudio.com/) and click the download button for either the .deb (Ubuntu, Debian) or the .rpm (Fedora, CentOS) file.
    Double-click the downloaded file to install VSCode. (You may be prompted to type your administrator password during the install).
```

**Pull request merged**